clang/lib/Basic/Targets/SystemZ.h

```diff
@@ -119,6 +119,10 @@ class LLVM_LIBRARY_VISIBILITY SystemZTargetInfo : public TargetInfo {
                              TargetInfo::ConstraintInfo &info) const override;
 
   std::string convertConstraint(const char *&Constraint) const override {
+    if (llvm::StringRef(Constraint) == "@cc") {
+      Constraint += 2;
+      return std::string("{@cc}");
+    }
```

Comment:
I think we also should have a `case '@':` in the switch statement below and move this check there.